vllm/v1/attention/backends/flashinfer.py

```diff
@@ -173,7 +173,10 @@ def get_supported_kernel_block_size() -> list[int | MultipleOf]:
         # Note: Not sure for all platforms,
         # but on Blackwell, only support a page size of
         # 16, 32, 64
-        return [16, 32, 64]
+        # TODO: 16 is temporary removed because TRT-LLM kernel has a bug when using 16.
```

Comment:
if the problem only exist in trtllm, what about only remove 16 for trtllm like
```
if current_platform.is_device_capability(100):
    return [32, 64]
else:
    return [16, 32, 64]
```

Reply:
Agreed, we should only override on Blackwell